### PR TITLE
[DO NOT MERGE] CHEBI import

### PR DESCRIPTION
**Pull request by shawntanzk**
Note: this PR is int base file imports to get CHEBI pulled in and CARO dangling classes removed

-CARO import seems to cause some error so I removed it (I think) 
-Currently tried `sh run.sh refresh_imports` (thought was to get the mirrors first) - ran into error, but noticed the mirrors seem to have been downloaded, so tried `sh run.sh make no-mirror-refresh-imports` and ran into error with go_import:

```
make[1]: *** [Makefile:246: imports/go_import.owl] Error 1
make[1]: Leaving directory '/work/src/ontology'
make: *** [Makefile:273: no-mirror-refresh-imports] Error 2
```

### CODE REVIEW

**Comment by shawntanzk** [comment]
```
annotate --ontology-iri http://purl.obolibrary.org/obo/pcl/imports/go_import.owl annotate -V http://purl.obolibrary.org/obo/pcl/releases/2021-12-13/imports/go_import.owl --annotation owl:versionInfo 2021-12-13 --output imports/go_import.owl.tmp.owl && mv imports/go_import.owl.tmp.owl imports/go_import.owl; fi
Exception in thread "main" java.lang.OutOfMemoryError: Java heap space
```

Memory issues :( will see if I can up the memory or something

**Comment by shawntanzk** [comment]
of note: I run into Error 18 when trying to make mirror/chebi.owl
with `robot_java_args: '-Xmx24G'` still facing the same issue

File: pcl-odk.yml [hunk 1]
```diff
@@ -25,4 +25,4 @@ import_group:
     - id: bfo
     - id: cl
     - id: omo
-robot_java_args: '-Xmx8G'
\ No newline at end of file
+robot_java_args: '-Xmx24G'
```
Comment (matentzn) [inline pcl-odk.yml:28]: Wrong file. src/ontology/pcl-odk.yml. Delete this one.

Also: don't forget update_repo

File: src/ontology/tmp/merged-pcl-edit.owl [hunk 2]
```diff
@@ -18,7 +18,10 @@
         <dc:contributor>https://orcid.org/0000-0001-7258-9596</dc:contributor>
```
Comment (matentzn) [inline src/ontology/tmp/merged-pcl-edit.owl:18]: remove from git.

File: src/ontology/tmp/pre_seed.txt [hunk 3]
```diff
@@ -58232,6 +58232,7 @@ http://purl.obolibrary.org/obo/UBERON_8440001
 http://purl.obolibrary.org/obo/UBERON_8440002
```
Comment (matentzn) [inline src/ontology/tmp/pre_seed.txt:58232]: remove from git

File: src/ontology/tmp/seed.txt [hunk 4]
```diff
@@ -58234,6 +58234,7 @@ http://purl.obolibrary.org/obo/UBERON_8440003
 http://purl.obolibrary.org/obo/pcl.owl
```
Comment (matentzn) [inline src/ontology/tmp/seed.txt:58234]: remove from git

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/tmp/merged-pcl-edit.owl (hunk 2 above)
Comment (matentzn) [reply src/ontology/tmp/merged-pcl-edit.owl:18]: (the file I mean)

File: pcl-odk.yml (hunk 1 above)
Comment (matentzn) [reply pcl-odk.yml:28]: 24 GB is _way_ too much. 8 is lower limit, try 12.

**Comment by shawntanzk** [comment]
`sh run.sh make no-mirror-refresh-imports` ran through without errors - however -edit file still has all the dangling terms -> will merge this into the other PR and try to fix it there

**Pull request merged**